lib/beaker-hostgenerator/data.rb

```diff
@@ -1160,10 +1160,7 @@ def generate_osinfo
       yield %w[aix73-POWER aix-7.3-power]
 
       # Fedora
-      (19..41).each do |release|
```

Comment:
As noted on IRC, there are official builds that still look maintained on Fedora 36.